server/api.coffee

```diff
@@ -0,0 +1,29 @@
+url = require 'url'
+
+apiMethods =
+  '/roomNew': (options) ->
+    try
+      result = Meteor.call 'roomNew',
+        grid: new Boolean options.grid
+      status: 200
+      json:
+        ok: true
+        id: result
+        url: Meteor.absoluteUrl "/r/#{id}"
```

Comment:
This is nice!  I had thought about doing this, but I wasn't sure how to get an external URL.

My question is how/if this plays nicely when cocreate sits behind an NGINX reverse proxy...how does `Meteor.absoluteUrl` actually know about the public-facing URL?  Is that in a configuration file somewhere?

Reply:
As [documented here](https://docs.meteor.com/api/core.html#Meteor-absoluteUrl), it's configured via the `ROOT_URL` environment variable. See for example [my configuration](https://github.com/edemaine/cocreate/blob/master/.deploy/mup.js) when deploying with [mup](http://meteor-up.com/). So a properly configured server will do this correctly.